tests/integration/telemetry/stats/prometheus/nullvm/dashboard_test.go

```diff
@@ -67,7 +67,8 @@ var dashboards = []struct {
 			// In default install, we have no proxy
 			"istio-proxy",
 			// https://github.com/istio/istio/issues/22674 this causes flaky tests
-			"galley_validation_passed",
+			//"galley_validation_passed",
+			"galley_validation_failed",
```

Comment:
Oddly, this test fails looking for `galley_validation_failed` on the primary cluster in the multicluster topology, even though all this PR is changing is making remote clusters, which are excluded (`requirePrimary == true`) from this test anyway, istiodless. The istio installation on the primary cluster, where it fails, is unchanged.

I also noticed that the test seems to work fine without excluding `galley_validation_passed` on both single and multicluster topologies. Maybe the bug is fixed?

@howardjohn @douglas-reid Any ideas? Is this change OK?

Reply:
clarification: it sounds like we want to remove `galley_validation_passed` despite it now seeming to work?  and we want to add `galley_validation_failed` despite it breaking the test?

Or have I misread that?

what in the test would cause a validation failure?

Reply:
ah, wait, i see. These are the excluded.

Reply:
This seems fine to me -- unless there is something that should be causing validation failures.

Reply:
RIght, before my change it seemed to be expecting a validation failure, but now there is none (at least in the multicluster env), so I had to exclude it. Seems strange that it was expecting some kind of failure anyway.

The validation_passed one was previously excluded because it was flaky according to @howardjohn, but after re-enabling it, I don't see any failures, so I wondering if the flakiness is fixed now?

Reply:
It only failed in postsubmit on old k8s versions. No clue why. So it may pass in presubmit and fail in postsubmit

Reply:
I guess it's worth trying to reenable it and see if the postsubmit failure returns. If it does I'll send a followup PR to exclude it again.